File: 2706.buy-two-chocolates.java
// @leet start
class Solution {
	public int buyChoco(int[] prices, int money) {
    int c1 = Integer.MAX_VALUE;
    int c2 = c1;
    for (int price : prices) {
      if (price < c1) {
        c2 = c1;
        c1 = price;
      } else if (price < c2) {
        c2 = price;
      }
    }
    if (c1 + c2 <= money) {
      return money - c1 - c2;
    }
    return money;
  }
}
// @leet end

File: 2706.buy-two-chocolates.js
// @leet start
/**
 * @param {number[]} prices
 * @param {number} money
 * @return {number}
 */
var buyChoco = function(prices, money) {
	let choc1 = Infinity;
	let choc2 = choc1;
	for (const price of prices) {
		if (price < choc1) {
			choc2 = choc1;
			choc1 = price;
		} else if (price < choc2) {
			choc2 = price;
		}
	}
	return money - choc1 - choc2 >= 0 ? money - choc1 - choc2 : money;
};
// @leet end
